Sources/Atomics/atomics-reference.swift

```diff
@@ -120,14 +108,10 @@ extension AtomicReference
   @inlinable
   public mutating func load(order: LoadMemoryOrder = .sequential) -> T?
   {
-    if let pointer = ptr.spinLoad(.lock, order)
+    if let pointer = ptr.load(order)
     {
-      assert(ptr.load(.sequential) == UnsafeRawPointer(bitPattern: 0x7))
-      let unmanaged = Unmanaged<T>.fromOpaque(pointer).retain()
-      // ensure the reference counting operation has occurred before unlocking,
-      // by performing our store operation with StoreMemoryOrder.release
-      ptr.store(pointer, .release)
-      return unmanaged.takeRetainedValue()
+//      assert(ptr.load(.sequential) == UnsafeRawPointer(bitPattern: 0x7))
```

Comment:
In the gap between line 111 and line 114, you have no way to prevent the reference count from going to zero.